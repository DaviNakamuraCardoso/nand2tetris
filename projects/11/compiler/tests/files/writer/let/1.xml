<keyword>let</keyword>
<identifier>b</identifier>
<symbol>[</symbol>
<integerConstant>23</integerConstant>
<symbol>]</symbol>
<symbol>=</symbol>
<integerConstant>17</integerConstant>
<symbol>;</symbol>
