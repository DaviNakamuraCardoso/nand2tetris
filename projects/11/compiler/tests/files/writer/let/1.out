push constant 23
push argument 0
add
push constant 17
pop temp 0
pop pointer 1
push temp 0
pop that 0
